dotcom-rendering/src/components/Card/components/AvatarContainer.tsx

```diff
@@ -26,7 +26,7 @@ const largerTopMargin = css`
 `;
 
 const sizingStyles = (
-	imageSize: ImageSizeType,
+	imageSize: MediaSizeType,
```

Comment:
Assuming imageSize and imagePositionType have been left as is as avatar is (currently) always an image.